website/docs/d/datasource_cloudfunctions_function.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "google"
+page_title: "Google: google_cloudfunctions_function"
+sidebar_current: "docs-google-datasource-cloudfunctions-function"
+description: |-
+  Get information about a Google Compute Image.
+---
+
+# google\_cloudfunctions\_function
+
+Get information about a Google Cloud Function.
```

Comment:
Let's also add the "for more information" line with a link to the GCP docs here just to be safe